NuGet/IronSoftware.Drawing.nuspec

```diff
@@ -39,21 +39,21 @@ Supports:
 
 For general support and technical inquiries, please email us at: support@ironsoftware.com</description>
 		<summary>IronSoftware.System.Drawing is an open-source solution for .NET developers to replace System.Drawing.Common with a universal and flexible library.</summary>
-		<releaseNotes>- Fixes incorrect HorizontalResolution and VerticalResolution of AnyBitmap images.
-		- Improves library's internal functionality and performance.</releaseNotes>
+		<releaseNotes>- Updates internal dependencies.
+    - Disable warning messages from BitMiracle.LibTiff.NET.</releaseNotes>
 		<copyright>Copyright © Iron Software 2022-2025</copyright>
 		<tags>Images, Bitmap, SkiaSharp, SixLabors, BitMiracle, Maui, SVG, TIFF, TIF, GIF, JPEG, PNG, Color, Rectangle, Drawing, C#, VB.NET, ASPX, create, render, generate, standard, netstandard2.0, core, netcore</tags>
 		<repository type="git" url="https://github.com/iron-software/IronSoftware.Drawing.Common" commit="$commit$" />
 		<dependencies>
 			<group targetFramework="netstandard2.0">
-				<dependency id="IronSoftware.Drawing.Abstractions" version="2025.2.6" />
+				<dependency id="IronSoftware.Drawing.Abstractions" version="2025.4.2" />
 				<dependency id="SixLabors.ImageSharp" version="2.1.10" />
 				<dependency id="SixLabors.ImageSharp.Drawing" version="1.0.0" />
 				<dependency id="BitMiracle.LibTiff.NET" version="2.4.649" />
 				<dependency id="System.Memory" version="4.5.5" />
 			</group>
 			<group targetFramework="net60">
-				<dependency id="IronSoftware.Drawing.Abstractions" version="2025.2.6" />
+				<dependency id="IronSoftware.Drawing.Abstractions" version="2025.4.2" />
 				<dependency id="SixLabors.ImageSharp" version="3.1.7" />
 				<dependency id="SixLabors.ImageSharp.Drawing" version="2.1.5" />
```

Comment:
Please update `SixLabors.ImageSharp.Drawing` to latest version "2.1.6" for net60 target. Please also change in .csproj as well.